common/src/main/java/org/bonitasoft/console/common/server/preferences/constants/WebBonitaConstantsImpl.java

```diff
@@ -102,7 +103,8 @@ public String getTenantTemplateFolderPath() {
     @Override
     public String getTempFolderPath() {
         if (tempFolderPath == null) {
-            tempFolderPath = getPlatformFolderPath() + tmpFolderName + File.separator;
+            tempFolderPath = System.getProperty("java.io.tmpdir") + File.separator + tmpFolderName + ManagementFactory.getRuntimeMXBean().getName()
```

Comment:
To have a tempFolder specific to the running JVM, so that 2 JVMs running on the same machine are isolated.